internals/network/concurrent_downloader.go

```diff
@@ -11,6 +11,9 @@ import (
 	pkg "github.com/arvyshka/blazer/pkg/data"
 )
 
+const (
```

Comment:
is it better to keep this const in globals.go ?

Reply:
I had the same thought then I looked for the best practices.
https://stackoverflow.com/a/6220792/4187022

Reply:
we will create an issue to remove that global.go file too